xml/templates/message_templates/participant_cancelled_text.tpl

```diff
@@ -2,6 +2,7 @@
 
 {ts}Your Event Registration has been cancelled.{/ts}
 
+{ts 1=$domain.phone 2=$domain.email}Please contact us at %1 or send email to %2 if you have questions.{/ts}
```

Comment:
This feels like something which should not be part of a stock template but should be up-to individual organisations to decide if they want.

Reply:
This line is already in the same mail. I just copied it to the top.